.github/workflows/app-release.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - 0.8.0-beta-fixes
```

Comment:
Maybe instead of providing branch name, add here (`workflow_dispatch`)[https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow]